app/views/includes/upscan_file_upload_form.scala.html

```diff
@@ -61,5 +62,15 @@
         attributes = Map("id" -> "submit")
     ))
 
-    <div class="govuk-!-display-none" id="progress-spinner"><img id="progress-spinner-img" src="@routes.Assets.versioned("images/spinner.gif")" alt="Loading ..."/></div>
+    <div class="govuk-!-display-none" id="progress-spinner">
+        <div class="govuk-!-margin-bottom-2 spinner">
+            <img id="progress-spinner-img" src='@routes.Assets.versioned("images/spinner.gif")'>
```

Comment:
Should have an Alt tag ?

Reply:
For this the Alt tag is made redundant as when the user clicks the upload button the focus is drawn to the new warning text element with the proper message